api/services/fatsecret_scraper/scraper.py

```
@@ -0,0 +1,220 @@
+
+        if not first_link:
+            raise Exception("No valid result found.")
+
+        # Dismiss cookie if needed
+        try:
+            driver.execute_script("""
+                const btn = document.querySelector('.cc-btn.cc-allow');
+                if (btn) { btn.click(); }
+            """)
+            sleep(1)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)